tests/acceptance/blocks/filterboxcomponent.js

```diff
@@ -0,0 +1,88 @@
+import { Selector, t } from 'testcafe';
+import FilterOptionsComponentBlock from './filteroptionscomponent';
+/**
+ * This class models user interactions with the {@link FilterBoxComponent}.
+ */
+export default class FilterBoxComponentBlock {
+  constructor (container) {
+    this._container = container || '';
+  }
+
+  /**
+   * Toggle the given option for the given filter.
+   */
+  async toggleOption (filterIndex, optionIndex) {
+    const filterOptions = await this.getFilter(filterIndex);
+    const isExpanded = await filterOptions.isExpanded();
+    if (isExpanded) {
+      await filterOptions.expand();
+    }
+    filterOptions.toggleOption(optionIndex);
+  }
+
+  /**
+   * Get the number of results associated with a particular option.
+   * This value is enclosed within parenthesis at the end of an option's
+   * label.
+   * @param {number} index
+   */
+  async getOptionCount (filterIndex, optionIndex) {
```

Comment:
I don't think we need this convenience method. It's mixing levels of abstraction. Filterbox can give you a FilterOptions and from FilterOptions you can get this count.